addons/pt/saft/extensions.go

```diff
@@ -420,6 +603,73 @@ var extensions = []*cbc.Definition{
 			i18n.EN: "Payment Means",
 			i18n.PT: "Meio de Pagamento",
 		},
+		Desc: i18n.String{
+			i18n.EN: here.Doc(`
+				The SAF-T's ~PaymentMechanism~ (Meios de pagamento) field specifies the payment means in a
+				sales invoice or payment. GOBL provides the ~pt-saft-payment-means~ extension to set this
+				value in your ~bill.Invoice~ advances or in you ~bill.Receipt~ method. GOBL maps certain
+				payment mean keys automatically to this extension:
+
+				| Code | Name                                               | GOBL Payment Mean                                     |
```

Comment:
So this table for example probably does make sense as there is a specific mapping from keys to types... is it worth considering reversing the order? Ideally you'd want to know what Payment Means `key` to use and the resulting code.